### DISCUSSION TITLE
Estructura JSON request

### DISCUSSION THREAD

**Discussion by jparadaa** [AndresRamos/ARSoftware.Contpaqi.Comercial.Api#6] [Q&A] [answered]
Si hay datos que no voy a enviar los debo de dejar null o vacíos o no ponerlos, por ejemplo:

"concepto": {
                "codigo": "101",
                "nombre": "",
                "datosExtra": {}

o 

"concepto": {
                "código": "101",

Saludos,
Javier

**Comment by AndresRamos** [comment, marked as answer]
Puedes dejar los valores en `null` o `""` dependiendo de si el valor puede ser nulo o no. Pero Tambien puedes omitirlo de la estructura JSON y se asignaran sus valores predeterminados. 

En tu caso, que creo que debe ser referente a la creación de un documento, puedes usar la segunda opción de no agregar los campos ya que solo el concepto es obligatorio y generarías una estructura más clara y leíble. Pero como te explico, no es un requerimiento que incluyas los campos si son opcionales.

Agregare documentación al Wiki para hacer más claro este requerimiento.
